arbeitsplan/templates/home.html

```diff
@@ -12,191 +12,243 @@
 
 {% if msg %}
 <div class="alert alert-{{msgclass}}">
-{{ msg }}
+  {{ msg }}
 </div>
 <p>
-{% endif %}
+  {% endif %}
+
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'custom_css/svpb.css'%}" />
```

Comment:
Stylesheets should be loaded inside HTML `<head>`, but the block content is inside `<body>`.
We have an extrastyle block (see https://github.com/svpbde/svpb/blob/630eca0d0099362a74ded707a61cc3d615c277a4/templates/base.html#L21), but currently it's used to define the background image...
Considering that in the base template we also include comparingly big libraries not used on every page (select2-bootstrap and datepicker), I think it would be fine to just place the custom css directly in the base template.